modules/applications/pages/ui-builder/download-and-launch.adoc

```diff
@@ -1,18 +1,34 @@
 = Download and launch Bonita UI Builder
 :page-aliases: applications:download-and-launch.adoc
-:description: Describes how to download and launch Bonita UI Builder.
+:description: Lean how to setup Bonita UI Builder environment ready for development.
 
 [NOTE]
 ====
 For Subscription editions only.
 ====
 
+{description}
+
 == Prerequisites
-* Docker installed
-* 2024.3 Bonita (or any later version) started on the local machine
+* Docker installed.
+* 2024.3 Bonita (or any later version) started on your local machine.
+* Ensure no firewall is blocking the connection between Bonita UI Builder and Bonita Runtime. Firewall configuration depends on your operating system.
```

Comment:
```suggestion
```

Reply:
@marcobonita why do you suggest to remove this (essential) sentence?